src/registrar/management/commands/load_transition_domain.py

```diff
@@ -0,0 +1,463 @@
+        # we have (count occurences)
+
+        to_create = []
+
+        # keep track of statuses that don't match our available
+        # status values
+        outlier_statuses = []
+
+        # keep track of domains that have no known status
+        domains_without_status = []
+
+        # keep track of users that have no e-mails
+        users_without_email = []
+
+        # keep track of duplications..
+        duplicate_domains = []
+        duplicate_domain_user_combos = []
+
+        # keep track of domains we ADD or UPDATE
+        total_updated_domain_entries = 0
+        total_new_entries = 0
+
+        # if we are limiting our parse (for testing purposes, keep
+        # track of total rows parsed)
+        total_rows_parsed = 0
+
+        logger.info("Reading domain-contacts data file %s", domain_contacts_filename)
+        with open(domain_contacts_filename, "r") as domain_contacts_file:
+            for row in csv.reader(domain_contacts_file, delimiter=sep):
+                total_rows_parsed += 1
+
+                # fields are just domain, userid, role
+                # lowercase the domain names
+                new_entry_domainName = row[0].lower()
+                userId = row[1]
+
+                new_entry_status = TransitionDomain.StatusChoices.READY
+                new_entry_email = ""
+                new_entry_emailSent = False
+                # TODO: how to know if e-mail was sent?
+
+                if new_entry_domainName not in domain_status_dictionary:
+                    # this domain has no status...default to "Create"
+                    if new_entry_domainName not in domains_without_status:
+                        domains_without_status.append(new_entry_domainName)
+                else:
+                    original_status = domain_status_dictionary[new_entry_domainName]
+                    # print(originalStatus)
+                    mapped_status = self.get_mapped_status(original_status)
+                    if mapped_status is None:
+                        logger.info("Unknown status: " + original_status)
+                        outlier_statuses.append(original_status)
+                    else:
+                        new_entry_status = mapped_status
+
+                if userId not in user_emails_dictionary:
+                    # this user has no e-mail...this should never happen
+                    if userId not in users_without_email:
+                        users_without_email.append(userId)
+                else:
+                    new_entry_email = user_emails_dictionary[userId]
+
+                # Check for duplicate data in the file we are
+                # parsing so we do not add duplicates
+                # NOTE: Currently, we allow duplicate domains,
+                # but not duplicate domain-user pairs.
+                # However, track duplicate domains for now,
+                # since we are still deciding on whether
```

Comment:
Agreed btw that we should allow duplicate domains and just not a duplicate user/domain pairing. This this is still the right approach